### DISCUSSION TITLE
For u

### DISCUSSION THREAD

**Discussion by BlackSparkz** [zyrouge/symphony#169] [General]
Are you left handed ?

**Comment by zyrouge** [comment]
Nope. Why though? Quite curious why you asked.

**Reply by BlackSparkz** [reply]
Because your application is more comfortable for left handed people
Most of user are right handed so plz optimise application for right handed i mean navigation buttons ( ◀️⏸️▶️ ) put on right side it's more comfortable for one hand use and short hand people

Thanku

I'm not good in English sorry 😔

**Reply by zyrouge** [reply]
You could change the layout in settings.

**Reply by BlackSparkz** [reply]
Only 2 option default one and traditional both are you not comfortable for me plz make traditional player like Spotify or yt music put navigation buttons slightly uper side from smartphone chin

**Reply by zyrouge** [reply]
Could you possible make a mockup of that? The traditional layout is very much customisable and does look like any generic ones.

**Reply by BlackSparkz** [reply]
![20230529_200357.jpg](https://github.com/zyrouge/symphony/assets/78287172/4b191839-82cf-4306-b5cc-5b16de1d3b6f)

![20230529_200959.jpg](https://github.com/zyrouge/symphony/assets/78287172/93016e83-e794-492f-a737-7f79e1684bf7)

See the position of buttons

**Reply by zyrouge** [reply]
The mockup doesn't quite look good. The bottom space is just too much. Maybe you need the slider above those buttons instead?

**Reply by BlackSparkz** [reply]
We will not be able to explain much just copy the now player of spotify in traditional layout and make layout in default like screenshot

**Reply by zyrouge** [reply]
Specifying what you actually need is better. I'm against literally copying something from other apps. Also, I'll highly recommend opening an issue for this.

**Reply by BlackSparkz** [reply]
As you wish it's okay 👍 and what do you think about about this ![20230529_200959.jpg](https://github.com/zyrouge/symphony/assets/78287172/a1919072-5bdc-4b11-af04-8f47f02420c9)

**Reply by zyrouge** [reply]
That looks awful as well.

**Reply by BlackSparkz** [reply]
It's more comfortable as compare to default

**Reply by zyrouge** [reply]
I'd suggest you to stick with traditional layout, that's the best solution right here. Unless you mockup promotes both UI and UX, it's probably not going to be implemented.

**Reply by BlackSparkz** [reply]
How to promote i don't know about this 
Mean i create issue

**Reply by BlackSparkz** [reply]
Actually I'm not saying copy the whole app ui just copy the position of buttons and all elements of now player tab

**Reply by zyrouge** [reply]
> How to promote i don't know about this Mean i create issue

Using the issues tab.
